python/paddle/fluid/tests/unittests/test_imperative_auto_mixed_precision.py

```diff
@@ -165,6 +165,34 @@ def run_simple_conv(inp_np, use_scaler=True):
                 np.allclose(outs_with_scaler[1][i][0].numpy(),
                             outs_no_scaler[1][i][0].numpy()), True)
 
+    def test_lamb_grad_clip(self):
+        inp_np = np.random.random(size=[1, 3, 128, 128]).astype(np.float32)
+
+        def run_simple_conv(inp_np, use_scaler=True):
+            paddle.seed(10)
+            paddle.framework.random._manual_program_seed(10)
+            with fluid.dygraph.guard():
+                model = SimpleConv(
+                    num_channels=3,
+                    num_filters=64,
+                    filter_size=7,
+                    stride=2,
+                    act='relu')
+                optimizer = paddle.optimizer.Lamb(
+                    learning_rate=0.01, parameters=model.parameters())
+                scaler = fluid.dygraph.AmpScaler(init_loss_scaling=1024)
+                data = fluid.dygraph.to_variable(inp_np)
```

Comment:
Same to those fluid APIs, should we use Paddle 2.0 API?

Reply:
Thanks, this unittest had be deleted.